### DISCUSSION TITLE
I want to add the @media screen as a builtin modifier

### DISCUSSION THREAD

**Discussion by coravacav** [tailwindlabs/tailwindcss#11624] [Ideas]
I'm writing some things with `prose`, and that's going great. The thing is though, it doesn't look so great in a print view. I think it'd be useful (and relatively easy) to add a `screen:` modifier to explicitly add the 
```css
@media screen
``` 
to the relevant css. This would allow me to only add the prose on non printing (ie `screen:prose`)

I'm happy to contribute it, but, the guide says ask here first so I figured I would :)

**Comment by mybearworld** [comment]
You could use `prose print:not-prose`.
